File: bernardo/Problema0247/problema0247.java
import java.util.Scanner;

public class problema0247 {
    static Scanner input = new Scanner(System.in);

    public static void main(String[] args) {

        // Declaração de variáveis
        float matrizA[] = new float[20];
        float matrizB[] = new float[20];
        int tamanho = 20;

        // Entrada de valores
        for (int i = 0; i < tamanho; i++) {
            System.out.println("Digite o " + (i + 1) + " número: ");
            matrizA[i] = Float.valueOf(input.nextLine());
        }

        // Cálculo da matrizB
        for (int i = 0; i < tamanho; i++) {
            matrizB[i] = matrizA[tamanho - 1 - i];
        }

        // Saída
        System.out.println("\nMatrizA:\n");
        for (int i = 0; i < tamanho; i++) {
            System.out.println(matrizA[i]);
        }
        System.out.println("\nMatrizB:\n");
        for (int i = 0; i < tamanho; i++) {
            System.out.println(matrizB[i]);
        }
    }
}

File: bernardo/Problema0247/problema0247.py
# Entrada de valores
matrizA = []
matrizB = []
tamanho = 20

for num in range(tamanho):
    matrizA.append(float(input(f"Digite o {num + 1} número: ")))

# Cálculo da matrizB
for num in range(tamanho):
    matrizB.append(matrizA[tamanho - 1 - num])

# Saída
print(f"MatrizA:\n {matrizA}")
print(f"MatrizB:\n {matrizB}")
